acra-censor/acra-censor_test.go

```diff
@@ -414,7 +414,7 @@ func testWhitelistColumnsPattern(t *testing.T) {
 	acceptableQueries = []string{
 		"SELECT a1 FROM table1 GROUP BY a2 HAVING COUNT(a3) > 0",
 		"SELECT a1 FROM table1 GROUP BY a2 HAVING COUNT(a2) > 1000",
-		"SELECT a1 FROM table1 GROUP BY a2 HAVING COUNT(a1) > (select 1)",
```

Comment:
Because we decided some time ago, that we do not consider query as value (https://github.com/cossacklabs/acra/blob/master/acra-censor/handlers/handlers_util.go#L677)